frontend/justfile

```diff
@@ -71,3 +71,19 @@ run *args:
 
 types:
     cd .. && pnpm exec vue-tsc -p frontend --noEmit
+
+##########
+# Health #
+##########
+
+# Check the health of the service
+@health host="localhost:50288":
+    -curl -s -o /dev/null -w '%{http_code}' 'http://{{ host }}/api/health'
```

Comment:
Can you change the name of this to `health-plausible` so that it is clear _which_ service it is checking (specifically, that it isn't checking for the "frontend" service, which doesn't exist).